CMakeLists.txt

```diff
@@ -137,11 +138,16 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
   add_definitions(-DDARWIN)
 endif(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 
-# Find/build external dependancies if it is Microsoft Visual Studio build
+# Find/build external dependencies if it is Microsoft Visual Studio build
 if(MSVC)
   find_package(Iconv REQUIRED)
   set(EXTERNAL_ICONV ${Iconv_LIBRARY})
+endif(MSVC)
 
+# Locate fontconfig, somehow the find_package call can't find it?
+include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/FindFontconfig.cmake)
```

Comment:
@maxirmx I don't understand -- why does this line work, but the `find_package(Fontconfig)` earlier did not work?

Reply:
find_package(XXX) is in reality a call to FindXXX.cmake   that is supposed to set several variables that define include, library paths and other options

very often source and binary packages provide FindXXX.cmake.   However, fontconfig does not provide FindFontconfig.cmake as a part of installation package. So people just include this cmake file developed by someone

Reply:
Ah, now I understand
It requires `set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake/")` otherwise it does not know where to look for cmake functions